solutions/remotemonitoring/single-vm/docker-compose.dotnet.yml

```diff
@@ -95,6 +95,11 @@ services:
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
       - PCS_APPLICATION_SECRET
+      - PCS_AAD_TENANT
```

Comment:
We will also need to change local deployment .yml files and may be documentation as well. Do you have a task to track that?

Reply:
No AAD application was created in local deployment.

Reply:
So when TSI gets deployed as part of local deployment, what does the developer experience look like?

Reply:
The Readme.md is pointing to this doc: https://docs.microsoft.com/en-us/azure/iot-accelerators/iot-accelerators-remote-monitoring-deploy-local. we may need some work there.

Reply:
Lets track this item that needs doc update, so we can engage @dominicbetts sooner. @asdonald Can help drive it.